client/wfnews-war/src/main/angular/src/app/components/report-of-fire/complex-question-page/rof-complex-question-page.component.scss

```diff
@@ -2,7 +2,7 @@
 
 ::ng-deep.mat-button-toggle-label-content {
   line-height: 35px !important;
-  padding-bottom: 1.5rem !important
+  padding-bottom: 3% !important;
```

Comment:
Are we sure a percentage is good here? Remember that mean it'll grow with screen size differences

Reply:
Changed to fixed size